### DISCUSSION TITLE
2024/07/07/user-group-001

### DISCUSSION THREAD

**Discussion by giscus** [wanghenshui/wanghenshui.github.io#134] [Q&A]
# 2024/07/07/user-group-001

https://wanghenshui.github.io/2024/07/07/user-group-001.html

**Comment by mapleFU** [comment]
感觉按照 https://cse.buffalo.edu/~zzhao35/teaching/cse707_fall21/l4-simd_saurabh.pdf 提的 Range 有点优化，但论文后面提到是计算占瓶颈才行，而且可能依赖一些手搓的 gather / scatter api。算 Bucket 重可能有优化

**Reply by wanghenshui** [reply]
嗯，分片重这么玩受益明显
